slog-extlog-derive/src/lib.rs

```diff
@@ -697,7 +724,10 @@ fn is_attr_stat_id(attr: &syn::Attribute, id: &syn::Ident) -> bool {
 }
 
 // Parses the StatTrigger attribute.
-fn parse_stat_trigger(attr_val: &[syn::NestedMetaItem], body: &syn::Body) -> StatTriggerData {
+fn parse_stat_trigger<'a, T>(attr_val: T, body: &syn::Data) -> StatTriggerData
```

Comment:
I'd have a mild preference for using `impl Iterator<...>` here as slightly more concise (and we don't really care what T is beyond that) over adding a generic parameter, but maybe that doesn't work with the lifetime?

Reply:
that works, I'll change